lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -467,7 +467,7 @@ def relationships_for(serializer, requested_associations, include_slice)
       #   }.reject! {|_,v| v.nil? }
       def links_for(serializer)
         serializer._links.each_with_object({}) do |(name, value), hash|
-          result = Link.new(serializer, value).as_json
+          result = Link.new(serializer, value, instance_options).as_json
```

Comment:
should probably mirror usage in pagination links?

- https://github.com/f-mer/active_model_serializers/blob/a6a8f53cadc1205b19f0ead41ab5c3f2d03bed10/lib/active_model_serializers/adapter/json_api/pagination_links.rb
- https://github.com/f-mer/active_model_serializers/blob/a6a8f53cadc1205b19f0ead41ab5c3f2d03bed10/lib/active_model_serializers/adapter/json_api.rb#L507